server/models/classes.py

```diff
@@ -0,0 +1,137 @@
+import requests
+from psycopg2 import Error,  pool
+
+devices_address_range = [
+    'http://localhost:5005',
+    'http://localhost:5004'
+]
+
+
+class Lokals():
+
+    def __init__(self):
+        self.apartments = []
+        self.load_apartment_from_database()
+
+    def create_apartments(self, name):
+        self.apartments.append(Apartment(name))
+
+    def load_apartment_from_database(self):
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute('SELECT apartment_name FROM apartments')
+            apartments_data = cursor.fetchall()
+            for apartment in apartments_data:
+                self.create_apartments(apartment[0])
+
+
+class Apartment:
+
+    def __init__(self, name):
+        self.name = name
+        self.rooms = []
+
+    def create_room(self, name):
+        self.rooms.append(Room(name))
+
+    def load_rooms(self):

```

Comment:
Don't load rooms directly from DataBase. Create another class to manage DB, and inside it - methods to get you, what you want.